.github/workflows/unittests.yaml

```diff
@@ -43,3 +43,10 @@ jobs:
       - name: Unittest and Coverage Report
         run: |
           python run_tests.py
+      # Provide code coverage reports on Linux
+      - if: ${{ matrix.os == 'ubuntu-latest' }}
+        name: coveralls.io
+        uses: AndreMiras/coveralls-python-action@develop
+        with:
+          # coveralls repo token
+          github-token: "SmlfzlVJy4ow55rduU7IU5GmmFCfAdGeq"
```

Comment:
I think we should be able to use a GitHub [encrypted secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow) to pass this along, and then this could become something like `github-token: ${{ secrets.coveralls_repository_token }}`

Reply:
> Note: With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.

which is stopping me from implementing. I'd like to see the coveralls report when builds are triggered by PRs from forked repos

Reply:
Sigh, nothing is ever easy with computers :(  That's an annoying limitation, makes sense to use the fixed token (I hope it's only valid for use with this repository on the Coveralls side).